.github/workflows/acceptance.yml

```diff
@@ -14,7 +22,39 @@ on:
   pull_request:
 
 jobs:
+  changes:
+    runs-on: ubuntu-18.04
+    # Set job outputs to values from filter step
+    outputs:
+      backend_or_frontend: ${{ steps.filter.outputs.backend_or_frontend }}
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout sentry
+
+      - name: Check for FE or BE file changes
+        uses: getsentry/paths-filter@v2
+        id: filter
+        with:
+          token: ${{ github.token }}
+          filters: .github/file-filters.yml
+
+  # It is necessary to update the name and matrix in tandem with the frontend workflow
+  # to meet the required check
+  fake-frontend:
+    if: needs.changes.outputs.backend_or_frontend != 'true'
+    needs: changes
+    runs-on: ubuntu-18.04
+    name: frontend tests
+    strategy:
+      matrix:
+        # XXX: When updating this, make sure you also update CI_NODE_TOTAL.
```

Comment:
Not needed here, however, we should add a comment in frontend job that this needs to be updated to match.